### PR TITLE
Remote branch test

### PR DESCRIPTION
**Pull request by darshanchavan1211**
Merging of remote-branch-test in main

### CODE REVIEW

File: ec2.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import boto3
+
+ec2 = boto3.client('ec2')
+ec2_res = ec2.describe_instances()
+# print("ec2 responce is :",ec2_res)
+# instance_id = ec2_res["Reservations"][0]["Instances"][0]["InstanceId"]
+key_n = ec2_res["Reservations"][0]["Instances"][0]["Tags"][0]["Key"]
+value_n = ec2_res["Reservations"][0]["Instances"][0]["Tags"][0]["Value"]
+print("key name is:",key_n)
```
Comment (awsdevopskiran) [inline ec2.py:9]: App print message properly it doesn't seems good

**Review by awsdevopskiran — COMMENTED** [review]
Made recommended chages

**Review by awsdevopskiran — CHANGES_REQUESTED** [review]
added review on the code

File: ec2.py (hunk 1 above)
Comment (darshanchavan1211) [reply ec2.py]: done with the changes

**Review by awsdevopskiran — APPROVED** [review]
Changes are fine

**Review by awsdevopskiran — COMMENTED** [review]
Changes looks fine

**Pull request merged**